### PR TITLE
Bug 1206597 - [RTL] corrects radio button split bar alignment for hom…

### PR DESCRIPTION
**Pull request by scottwu**
…escreen and APN list

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=429742fff2911a9b3dd35c7a5c6b1549fcb3946d

File: apps/settings/style/settings.css [hunk 1]
```diff
@@ -1124,7 +1124,9 @@ html[dir="rtl"] .apn-list .apn-item .name {
   right: -1.5rem;
 }
 
-html[dir="rtl"] .apn-list .apn-item:after {
+html[dir="rtl"] .apn-list .apn-item:after,
```
Comment (gasolin) [inline apps/settings/style/settings.css]: If the style rule is not valid anymore, please remove it

Comment (scottwu) [inline apps/settings/style/settings.css]: Oh yeah. Thanks for catching that

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=056dc61816dfb9dd3799075ca499e2f8972cbcbc

**Pull request merged**